factory.go

```diff
@@ -4,25 +4,14 @@ import (
 	"context"
 )
 
-// Factory provides context aware loggers.
-type Factory struct {
-	baseLogger Logger
-}
-
-// NewFactory instantiates a factory with the default logger.
-func NewFactory() Factory {
-	return Factory{
-		baseLogger: DefaultLogger,
-	}
-}
+var DefaultFactory Factory = NewFactory()
```

Comment:
There's probabl ya linter taht will ask you for some comments.

Reply:
done https://github.com/cabify/go-logging/pull/6/commits/2e9d2a5a1d52e11eb773b1db1fe053bae7db7138